kinesis-video-native-build/install-script

```diff
@@ -103,14 +103,14 @@ if [[ $build_producer == TRUE ]]; then
   echo "Checking openssl at $DOWNLOADS/local/lib/libssl.a"
   if [ ! -f $DOWNLOADS/local/lib/libssl.a ]; then
     echo "openssl lib not found. Installing"
-    if [ ! -f $DOWNLOADS/openssl-1.1.1d.tar.gz ]; then
+    if [ ! -f $DOWNLOADS/openssl-1.1.1f.tar.gz ]; then
       cd $DOWNLOADS
       #download_cmd="curl -L "https://www.openssl.org/source/openssl-1.1.1d.tar.gz" -o "openssl-1.1.1d.tar.gz" "
```

Comment:
Thank you for your contribution @kumpelj ! One small change and you are good to go. 

Can you change the comment to reflect the version?